extra/man/alacritty.5.scd

```diff
@@ -740,7 +740,8 @@ post_processing = _true_++
 persist         = _false_++
 mouse.enabled   = _true_++
 binding         = { key = _"O"_, mods = _"Control|Shift"_ }++
-regex = _"(ipfs:|ipns:|magnet:|mailto:|gemini://|gopher://|https://|http://|news:|file:|git://|ssh:|ftp://)[^\\u0000-\\u001F\\u007F-\\u009F<>\\"\\\\s{-}\\\\^⟨⟩`]+"_
+regex =
+_"(ipfs:|ipns:|magnet:|mailto:|gemini://|gopher://|https://|http://|news:|file:|git://|ssh:|ftp://)[^\\u0000-\\u001F\\u007F-\\u009F<>\\"\\\\s{-}\\\\^⟨⟩`\\\\\\\\]+"_
```

Comment:
@kchibisov Do you remember how we fixed the `~` in manpages? I just realized that the \` here is automatically transformed into a different character, which would likely cause problems.

Reply:
I've sent a patch to scdoc to explicitly use a special sequence once you face such a character, but not sure it made into any release.